### DISCUSSION TITLE
Add option to sample/reduce data points in Monitoring

### DISCUSSION THREAD

**Discussion by Christopher-Hayes** [vercel/community#6373] [Feature Question] [answered]
### Question

Right now, Monitoring will use 100% of data points (based on requests). For larger sites this usage cost can get extremely expensive.

**Feature Request:** Add a sampling option to Monitoring so users can pay a reduced usage cost for fewer analytics.

With Speed Insights we're given the option of sampling data points, so only X% of data points are actually collected and counted towards usage billing. If Speed Insights didn't have this, we wouldn't be paying for that add-on at all.

We're in the same situation with Monitoring, for certain use-cases it provides some very helpful information, but it's hard to tell the client that we're going to be spending $300 / month just so engineering can have a few more analytics. At scale, the usage cost doesn't justify the value proposition. We're going to have to disable the Monitoring add-on and wait for Vercel to making the usage billing more manageable.

**Comment by amyegan** [comment, marked as answer]
Good question @Christopher-Hayes. Sampling isn't currently an option with Monitoring, but this is a nice idea. I'm sharing it with the team for future consideration.
